benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj

```diff
@@ -4,9 +4,9 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp3.1;net472</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp3.1</TargetFrameworks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <LangVersion>8</LangVersion>
-    
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <LangVersion>latest</LangVersion>
```

Comment:
Benchmark run failed with error:

```
Standard error: 
 D:\a\1\s\benchmarks\Benchmarks.Trace\bin\Release\netcoreapp3.1\f04c6b44-ebed-47dc-8807-1a41f056cf15\BenchmarkDotNet.Autogenerated.csproj(20,18): error MSB4086: A numeric comparison was attempted on "$(LangVersion)" that evaluates to "latest" instead of a number, in condition "'$(LangVersion)' == '' Or '$(LangVersion)' < '7.3'".
```

I guess benchmarkdotnet doesn't support setting "latest" for version :/

Reply:
Nope, is not supported, that's why I had to forced to something previously.

Reply:
fixed in 518d68a5